examples/doa_algorithms_infiniteroom.py

```diff
@@ -0,0 +1,114 @@
+'''
```

Comment:
For that one, no need to create a new example, just replace `ShoeBox` in the original script by the new class. The original uses an anechoic room anyway.

Reply:
Great, I was going to do that, just kept both examples for now to have a quick way of comparing